packages/eth-json-rpc-provider/src/index.test.ts

```diff
@@ -4,6 +4,7 @@ describe('Package exports', () => {
   it('has expected exports', () => {
     expect(Object.keys(allExports)).toMatchInlineSnapshot(`
       Array [
+        "SafeEventEmitterProvider",
```

Comment:
What's the reason for this?

Reply:
This one follows from the export statement change for `SafeEventEmitterProvider` https://github.com/MetaMask/core/pull/1738/files#r1357548245. The test is checking what non-types are being exported from `index.ts`.